bff-node/src/middleware/modelValidation.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, NextFunction } from "express";
+import { AnyZodObject } from "zod";
+import { ServerCode } from "../enums/ServerCode";
+
+// Middleware to validate the request body, query and params against a schema
```

Comment:
Great! Is smart to have a middleware to validate incoming requests instead of validating it in the controller. 👏